src/lesson4Array2/Array2.java

```diff
@@ -0,0 +1,63 @@
+package lesson4Array2;
+
+import java.util.Scanner;
+
+public class Array2 {
+    private double length;
+    private double[] dataTable;
+    private static Scanner scanner = new Scanner(System.in);
+
+    public Array2(double length) {
+        this.length = length;
+        this.dataTable = new double[(int) length];
+    }
+
+    public int setData() {
+        for (double i = 0; i < dataTable.length; i++) {
+            System.out.println("Enter elements: " + i);
+            dataTable[(int) i] = scanner.nextDouble();
+        }
+        return 0;
+    }
+
+    public double calculateSum() {
+
+        if (dataTable == null) {
+            setData();
+            System.out.println("Enter you elements ");
+        } else {
+            double sum = 0;
+            for (double i = 0; i < dataTable.length; i++) {
+                sum += dataTable[(int) i];
+            }
+            return sum;
+        }
+        return 0;
+    }
+
+    public double calculateAverage() {
+        return calculateSum() / dataTable.length;
+    }
+
+    public double getHighest() {
+        double highest = dataTable[0];
+        for (double i = 0; i < dataTable.length; i++) {
```

Comment:
why double i??? The i is just an index. The index is always integer, could we have 1.5 index??? - No. Please, use int i = 0....